### PR TITLE
Update "software" method in Social

### PR DESCRIPTION
**Pull request by phallobst**
See #4410

Wykop (in Polish) must still be checked

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -270,7 +270,7 @@ websites:
     img: vk.png
     tfa:
       - sms
-      - software
+      - totp
     doc: https://vk.com/page-59800369_47885415
```
Comment (Carlgo11) [inline _data/social.yml]:
```suggestion
    doc: https://vk.com/page-777107_53677752
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
Let's remove the Wykop listing. I can't find anything about 2FA, registration requires a Polish number and their contact info is only in Polish (with a whois email that goes to Cyprus).

**Comment by phallobst** [comment]
I dug a little bit deeper... The Polish fork of 2FA at https://dwa-skladniki.pl/ has this as the doc:
> https://www.wykop.pl/wpis/24198791/wykopowicze-na-waszych-profilach-w-ustawieniach-ko/

It sounds like it's totp, since there are mentions of freeOTP and Authy. Shall we go for it?

**Review by Carlgo11 — APPROVED** [review]

**Review by Jawshy — APPROVED** [review]
All English sites ✅

**Pull request merged**